www/bala00/project_1/css/styles.css

```diff
@@ -0,0 +1,583 @@
+* {
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+    font-family: 'Quicksand', sans-serif;
+    font-weight: 400;
+}
+
+/* 
+#C1403D красный
+#90AFC5 нежно-голубой
+#336B87 голубой
+#2D4262 индиго
+#2A3132 серо-синий
+#363237 серый
+#763626 коричневый
+#73605B бежево-коричневый
+#D09683 персик
+*/
+
+/* common CSS */
+nav {
+    display: flex;
+    color: #C1403D;
+    margin: 10px;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    margin: 0;
+    padding: 0;
+}
+
+nav li {
+    padding: 0 14px;
+    line-height: 40px;
+}
+
+nav li a.active {
+    font-weight: 600;
+}
+
+nav a:link,
+a:link {
+    color: #C1403D;
+    text-decoration: none;
+}
+
+nav a:visited,
+a:visited {
+    color: #C1403D;
+    text-decoration: none;
+}
+
+nav a:hover,
+a:hover {
+    color: #C1403D;
+    text-decoration: none;
+}
+
+nav a:active,
+a:active {
+    color: #C1403D;
+    text-decoration: none;
+}
+
+footer {
+    text-align: center;
+    font-size: 50%;
+    line-height: 100%;
+    margin: 30px;
+}
+
+/* nepodarilo se mi posunout vyhledavaci tlacitko uplne vpravo, 
+   bez inputu to fungovalo dobre)) */
+.search input[type=text] {
+    width: 20px;
+    border: none;
+    box-sizing: border-box;
+    font-size: 16px;
+    letter-spacing: 1px;
+    font-weight: 300;
+    background-color: white;
+    background-image: url(../images/search.png);
+    background-position: 10px;
+    background-size: 20px;
+    padding: 10px 20px 10px 40px;
+    -webkit-transition: width 0.4s ease-in-out;
+    transition: width 0.4s ease-in-out;
+    /* margin-left: auto;
+    margin-inline-end: 5px; */
+    cursor: pointer;
+}
+
+.search input[type=text]:focus {
+    width: 100%;
+    outline: none;
+    font-weight: 700;
+}
+
+.logo-navbar img {
+    width: 40px;
+    height: 40px;
+}
+
+/* index.html */
+.intro {
+    height: 90vh;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    display: table;
+    background: url(../images/prague1.jpg) center center;
```

Comment:
tenhle effekt pro background se da vyfiltrovat pomoci CSS, nepouzivej dva obrazky kvuli tomu

Reply:
ten effekt pomoci CSS jsem zkousela, ale pak mi nefungovalo tlacitko, takze mam pouzit 'opacity', ale tak, aby to tlacitko pak fungovalo?) (nebo je na to neco jineho)

Reply:
ja bych to udelal tak, ze budou v podstate dve vrstvy, prvni vrstva bude mit background image,
druha vrstva bude mit background-color (bila pruhledna), a na teto vrstve budou text a tlacitky

Reply:
dobre, dekuji!